data/sea/54-zebra.h

```diff
@@ -156,7 +195,7 @@ static int64_t zebra_translate_column
                 int offset = 1;
                 for (int i = 0; i != count; ++i) {
                     int len = data.a.n[i];
-                    offset = zebra_translate_table (mempool, elem_start, len, (void**)(strings + i), &data.a.table);
+                    offset = zebra_translate_table (mempool, fact_offset, elem_start, len, (void**)(strings + i), &data.a.table);
```

Comment:
this was already half-wrong - it didn't support nested arrays properly which wasn't a problem, but now we need to slice out of the nested arrays

so you need to loop through from `[0,elem_start)` summing up the lengths and use that as `elem_start` for the recursive calls.
but this loop wouldn't be necessary after #473 since latest zebra stores them as prefix sum
anyway.

we might need another function to sum the offsets of a table without doing any translation - because at the moment if the array is empty the offset isn't being computed properly and just set to 1. that's not directly related to this PR but it's probably worth working this out at the same time if you want to go through it together tomorrow